package/src/constructs/browser-check.ts

```diff
@@ -2,7 +2,7 @@ import * as fs from 'fs'
 import * as path from 'path'
 import { Check, CheckProps } from './check'
 import { Session } from './project'
-import { Parser } from '../services/check-dependency-parser'
+import { Parser } from '../services/check-parser/parser'
```

Comment:
Nit: It could be handy to have an `index.ts` in `check-parser` that re-exports everything.

Reply:
I will most likely add that with the next PR